app/indices/cms_page_index.rb

```diff
@@ -1,7 +1,7 @@
 ThinkingSphinx::Index.define 'cms/page', with: :active_record do
   indexes :title
   has :tenant_id
-  where 'searchable=1'
+  where "searchable=#{ System::Database.postgres? ? 'TRUE' : '1' }"
```

Comment:
Maybe use `ActiveRecord::Type::Boolean`

Reply:
Refactored to use instead https://www.rubydoc.info/gems/thinking-sphinx/3.2.0/ThinkingSphinx%2FActiveRecord%2FInterpreter%3Asanitize_sql